### ISSUE TITLE
Main character disappears when global illumination is enabled

### ISSUE THREAD

**Issue by ebraminio** [godotengine/tps-demo#146]
**OS/device including version:**
Apple M1 Pro on latest version of Godot 3.5

**Issue description:**
The character disappears when Global Illumination is enabled.

**Screenshots of issue:**
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/833473/191953356-333d196c-d479-4c99-814d-3f812967d91f.png">

On

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/833473/191953415-0604bbf8-d3be-4845-a79c-2d416d8f91ed.png">

Both on High and Low option will not show anything. But the character shows up just if I disable global illumination, enabling or disabling other options seem to have no effect.

**Comment by aidan-j-rhoden** [comment]
I can mostly reproduce this on MacOS and a Linux emulator on Chromebook, both running Godot 3.5.stable.  However, for me, the character doesn't disappear; it completely turns black, along with the red robots.  Oddly enough, the light from the eyes of both robots are still visible on the black silhouette.

**Comment by ebraminio** [comment]
Thanks for confirming this, I went double checking this and found enemies also have got half transparent just like the main character only when global illumination is enabled, hopefully that helps you,

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/833473/191991878-ed74d835-074a-4eef-8fca-9a3921cbaf37.png">

**Comment by aaronfranke** [comment]
CC @Calinou

**Comment by Calinou** [comment]
Can you reproduce this in previous commits of the demo? If not, this is an engine issue.

**Comment by ebraminio** [comment]
Yes, reproduces with `git checkout c1c4cc2deed677f7abcb558713aaf7ee7b4be2ab` also, and noticed one extra thing, it only happens with GLES3 backend and not with GLES2 one.

**Comment by Calinou** [comment]
> and noticed one extra thing, it only happens with GLES3 backend and not with GLES2 one.

This is expected, as global illumination (GIProbe) is only supported with GLES3, not GLES2.
